### PR TITLE
Merge branch 'master' (1.3.0 content) into 'develop'

### PR DESCRIPTION
**Pull request by chrisvanheuveln**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,24 +27,8 @@ Changelog
 * Added support for bdi interfaces to interface provider.
 * Added a new node util to handle bridge domain range cli for member vni
 * Added Bridge Domain, VNI and encapsulation profile node utils for MT-FULL on Nexus 7k.
-* Added client support for gRPC on IOS XR.
```
Comment (mikewiebe) [inline CHANGELOG.md:30]: We need to add these XR references back into the changelog.

**Comment by chrisvanheuveln** [comment]
This is just a collapse back to develop.

File: README.md [hunk 2]
```diff
@@ -13,7 +13,6 @@ This workflow map aids *users*, *developers* and *maintainers* of the CiscoNodeU
   * [CONTRIBUTING.md](CONTRIBUTING.md) : Contribution guidelines
   * [README-develop-node-utils-APIs.md](docs/README-develop-node-utils-APIs.md) : Developing new CiscoNodeUtils APIs
   * [README-develop-best-practices.md](docs/README-develop-best-practices.md) : Development best practices
-  * [README-develop-add-XR-support.md](docs/README-develop-add-XR-support.md) : How To Update node_utils APIs to Support IOS XR
```
Comment (mikewiebe) [inline README.md:16]: Do you want to merge this and and then worry about merging the two main README docs?

File: docs/README-develop-add-XR-support.md [hunk 3]
```diff
@@ -1,277 +0,0 @@
-# How To Update node_utils APIs to Support IOS XR
```
Comment (mikewiebe) [inline docs/README-develop-add-XR-support.md:1]: This entire doc needs to be added back.

**Pull request merged**